recipe/meta.yaml

```diff
@@ -48,9 +48,12 @@ about:
   license: MIT
   license_file: LICENSE
   summary: Store and access your passwords safely
+  dev_url: https://github.com/jaraco/keyring
+  doc_url: https://pypi.org/project/keyring/
 
 extra:
   recipe-maintainers:
     - licode
     - tacaswell
     - ccordoba12
+    
```

Comment:
```suggestion
```

It seems you left an additional line here without a line ending, which is why the conda-forge linter complains about it. My suggestion above should fix it.